sso/api/views_activity_stream.py

```diff
@@ -162,3 +165,44 @@ def get(self, request, *args, **kwargs):
         }
 
         return Response(data=page)
+
+
+class ActivityStreamDirectorySSOUserAnswersVFM(ListAPIView):
+    authentication_classes = [_ActivityStreamAuthentication]
+    permission_classes = []
+
+    pagination_class = ActivityStreamDirectorySSOUsersPagination
+    queryset = UserAnswer.objects.all()
+    serializer_class = serializers.ActivityStreamUserAnswerSerializer
+
+    def get(self, request, *args, **kwargs):
+        queryset = self.filter_queryset(self.get_queryset())
+
+        queryset_page = self.paginate_queryset(queryset)
+        serializer = self.get_serializer(queryset_page, many=True)
+        page = {
+            '@context': [
+                'https://www.w3.org/ns/activitystreams',
+            ],
+            'type': 'Collection',
+            'orderedItems': [
+                {
+                    'dit:application': 'DirectorySSO',
```

Comment:
Nit: should the key be camelCase? so `directorySSO`?

```suggestion
                    'dit:application': 'directorySSO',
```

Reply:
just following the same pattern as other AS submissions not sure if it's standard syntax for AS.